slakh96/no-mans-land
# Added Main Menu

- Added MainMenuScreen canvas and MainMenuScript which implements the links and adds functionalities to the buttons 
TESTING INSTRUCTIONS: 
- In the Main Menu Scene, drag MainMenuCanvas to it 
- Drag DemoFinalScene into the hierarchy as well 
- Click on File -> Build Settings, and click "Add Open Scenes" 
- Make sure both MainManuScene (index = 0) and DemoFinalScene (index = 1) get added 
- Now close this window and try playing the game. Before playing, remove DemoFinalScene from the hierarchy
